src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || 'jujuda1993',
```

Comment:
Storing passwords directly in the code is a security risk. Consider using environment variables for the database password and ensure it is not hardcoded in the source code.